pkg/server/server.go

```diff
@@ -2,793 +2,38 @@ package server
 
```

Comment:
Now that I see this, the reason for the diff I commented on before makes more sense.

I would still like to preserve the git history of the applications `server.go` file. Maybe we can have a new file called `combined_server.go` (terrible name I know) that would represent the new mux'd apps+profiles server?